src/pages/AuthPage/Logo/Logo.styles.jsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const LogoWrapper = styled.div`
```

Comment:
Same comment as other PRs When you are naming a styled Component add the word Styled as a prefix. 

`StyledComponentName`